<launch>

  <node pkg="lslidar_c16_driver" type="lslidar_c16_driver_node" name="lslidar_c16_driver_node" output="screen">
    <param name="lidar_ip" value="192.168.1.222"/>
    <param name="device_port" value="2368"/>
    <param name="add_multicast" value="false"/>
    <param name="group_ip" value="224.1.1.2"/>
    <param name="pcap_file_path" value="/home/lyh/lab/data/iking0307/laser2020-3-7-15-31-12.pcap"/>
    <param name="start_time" value="27416"/>
    <param name="last_time" value="28256"/>
  </node>

  <node pkg="lslidar_c16_decoder" type="lslidar_c16_decoder_node" name="lslidar_c16_decoder_node" output="screen">
    <param name="frame_id" value="laser_link"/>
    <param name="point_num" value="2000"/>
    <param name="channel_num" value="0"/>
    <param name="angle_disable_min" value="0"/>
    <param name="angle_disable_max" value="0"/>
    <param name="angle3_disable_min" value="330"/>
    <param name="angle3_disable_max" value="210"/>
    <param name="min_range" value="20"/>
    <param name="max_range" value="1000.0"/>
    <param name="frequency" value="5"/>
    <param name="publish_point_cloud" value="true"/>
    <param name="publish_scan" value="true"/>
    <param name="use_gps_ts" value="true"/>
  </node>
  

  <!--node name="rviz" pkg="rviz" type="rviz" args="-d $(find lslidar_c16_decoder)/launch/lslidar_c16.rviz" output="screen"/-->

</launch>
